.github/workflows/containers.yaml

```
@@ -70,7 +70,14 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
-            GIT_REVISION=${{ steps.vars.outputs.revision }}
+            REACT_APP_TENANT_BASE_URL=https://api.rotational.app/v1/
+            REACT_APP_QUARTERDECK_BASE_URL=https://auth.rotational.app/v1/
+            REACT_APP_ANALYTICS_ID=${{ secrets.REACT_APP_ANALYTICS_ID }}
```

```suggestion
            REACT_APP_ANALYTICS_ID=${{ secrets.BEACON_ANALYTICS_ID }}
```

In case we have two UIs that need build variables. The envvars have to be prefixed with `REACT_APP` but the secret values.